_maps/map_files/generic/Admin_Level.dmm

```diff
@@ -750,6 +750,10 @@
 	icon_state = "47"
 	},
 /area/deathmatch)
+"cS" = (
+/obj/effect/ai_node,
```

Comment:
remove these, ai nodes were removed from eord arena for good reason

Reply:
If you meant admins spamming AI causing massive lag, see #5229

Reply:
I mean spawning them at all, players hate it.

Reply:
standing